### ISSUE TITLE
No getPoWHash error

### ISSUE THREAD

**Issue by Bushstar** [Crypto-Expert/stratum-mining#384]
Clean install, set up following the Wiki, when a miner gets a share we see the error.

> 'module' object has no attribute 'getPoWHash'

Stratum was installed as shown in the instructions from the following location but we see this issue I'm guessing with Scrypt coins.

https://github.com/ahmedbodi/stratum-mining

```
> 2017-10-13 15:52:28,669 ERROR protocol # [Failure instance: Traceback: <type 'exceptions.AttributeError'>: 'module' object has no attribute 'getPoWHash'
> /usr/local/lib/python2.7/dist-packages/stratum-0.3.14-py2.7.egg/stratum/protocol.py:192:dataReceived
> /usr/local/lib/python2.7/dist-packages/stratum-0.3.14-py2.7.egg/stratum/protocol.py:238:lineReceived
> /usr/local/lib/python2.7/dist-packages/stratum-0.3.14-py2.7.egg/stratum/services.py:13:_handle_event
> /usr/local/lib/python2.7/dist-packages/stratum-0.3.14-py2.7.egg/stratum/services.py:81:call
> --- <exception caught here> ---
> /usr/local/lib/python2.7/dist-packages/Twisted-17.9.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py:150:maybeDeferred
> /usr/local/lib/python2.7/dist-packages/stratum-0.3.14-py2.7.egg/stratum/services.py:78:_run
> /root/stratum-mining-4/./mining/service.py:166:submit
> /root/stratum-mining-4/./lib/template_registry.py:250:submit_share
> ]
> None
```

**Comment by Bushstar** [comment]
I think there an error in the sample config.

https://github.com/Crypto-Expert/stratum-mining/blob/master/conf/config_sample.py#L18

ALGO_NAME should be COINDAEMON_ALGO, changing that makes Scrypt load.

**Comment by ahmedbodi** [comment]
You need to specify ltc_scrypt as the algo

Sent from my iPhone

On 13 Oct 2017, at 17:16, Peter Bushnell <notifications@github.com<mailto:notifications@github.com>> wrote:


I think there an error in the sample config.

https://github.com/Crypto-Expert/stratum-mining/blob/master/conf/config_sample.py#L18

ALGO_NAME should be COINDAEMON_ALGO, changing that makes Scrypt load.

**Comment by Bushstar** [comment]
I ran...

    pip install scrypt

Mining works with algorithm set to just "Scrypt", a block had been minted on the pool.

Is there a difference between ltc_scrypt and Scrypt?

**Comment by ahmedbodi** [comment]
Weird might be so. I could be wrong but I thought I set it so you specify the module name rather than the algo name

Sent from my iPhone

On 14 Oct 2017, at 06:57, Peter Bushnell <notifications@github.com<mailto:notifications@github.com>> wrote:


I ran...

pip install scrypt


Mining works with algorithm set to just "Scrypt", a block had been minted on the pool.

Is there a difference between ltc_scrypt and Scrypt?
